src/components/molecules/OnlineStats/OnlineStats.tsx

```diff
@@ -74,7 +74,7 @@ const OnlineStats: React.FC = () => {
   >();
 
   const venue = useSelector(currentVenueSelectorData);
-  const partygoers = usePartygoers();
+  const { recentWorldUsers } = useRecentWorldUsers();
 
   const venueName = venue?.name;
```

Comment:
Venue name should not be optional as it's currently required in the venue creation process. Our typings are wrong. Leave this as a note..

Reply:
@denisdimitrov I think is actually about `venue` being potentially `undefined` here, but we are using the `?.name` to try and access `name` regardless.

It's basically shorthand for `const venueName = !!venue ? venue.name : undefined`, so I don't think our types are wrong in this instance (unless there is some other nuance I've missed?)

Reply:
Yeah my bad. Brain fart